src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewField.java

```diff
@@ -32,6 +34,8 @@
     private Boolean showSummaryChangeOption;
     @JsonProperty("show_summary_content_option")
     private Integer showSummaryContentOption;
+    @JsonProperty("acls")
```

Comment:
Shouldn't this field be ignored from the JSON payload? Otherwise it would go against the principle that users are not meant to know they don't have access to something. In that case, given the filtering is already performed server-side, I guess we don't need those ACLs client side anyway do we?

```suggestion
    @JsonIgnore
```